modules/migration-kubernetes-objects.adoc

```diff
@@ -5,9 +5,16 @@
 
 :_content-type: PROCEDURE
 [id="migration-kubernetes-objects_{context}"]
-= Migrating Kubernetes objects
+= Performing a state migration of Kubernetes objects by using the {mtc-short} API
 
-You can perform a one-time migration of Kubernetes objects that constitute an application's state.
+After you migrate all the PV data, you can use the {mtc-first} API to perform a one-time state migration of Kubernetes objects that constitute an application.
+
+You do this by configuring `MigPlan` custom resource (CR) fields to provide a list of Kubernetes resources with an additional label selector to further filter those resources, and then performing a migration by creating a `MigMigration` CR. The `MigPlan` resource is closed after the migration.
+
+[NOTE]
+====
+Selecting Kubernetes resources is an API-only feature. You must update the `MigPlan` CR and create a `MigMigration` CR for it via the CLI. The {mtc-short} web console does not support migrating Kubernetes objects.
```

Comment:
Not critical, but ISG says to avoid "via". You could replace it here with "by using"